src/main/java/requesthandler/UserLoginController.java

```diff
@@ -2,35 +2,31 @@
 
 import db.DataBase;
 import model.User;
-import webserver.controller.Controller;
+import webserver.controller.AbstractController;
 import webserver.request.Cookie;
 import webserver.request.HttpRequest;
 import webserver.response.HttpResponse;
 
+import static java.util.Objects.nonNull;
+
 /**
  * Created by hspark on 2019-08-05.
  */
-public class UserLoginController implements Controller {
+public class UserLoginController extends AbstractController {
     public static final String URL = "/user/login";
 
-
     @Override
-    public void action(HttpRequest httpRequest, HttpResponse httpResponse) {
-        String userId = httpRequest.getRequestBody().getOne("userId");
-        String password = httpRequest.getRequestBody().getOne("password");
+    public void doPost(HttpRequest httpRequest, HttpResponse httpResponse) {
+        String userId = httpRequest.getParameter("userId");
+        String password = httpRequest.getParameter("password");
 
         User user = DataBase.findUserById(userId);
-        if (user.matchPassword(password)) {
+        if (nonNull(user) && user.matchPassword(password)) {
             httpResponse.addCookie(new Cookie("logined", "true"));
```

Comment:
이전 pr에서 피드백 드렸던 내용이긴 한데요. https://github.com/next-step/jwp-was/pull/59#discussion_r310821338

```suggestion
            httpResponse.addCookie(new Cookie(COOKIE_LOGIN_KEY, String.valueOf(true)));
```
위와 같은 형태는 어떨까요? 
이후에 쿠키 값을 사용할 때 true/false 값의 명확성을 보장할 수 있을 것 같아요.